core/src/apps/webauthn/add_resident_credential.py

```diff
@@ -39,10 +39,9 @@ async def add_resident_credential(
         await show_error_and_raise(
             ctx,
             "warning_credential",
-            header="Import credential",
+            subheader="Import credential",
```

Comment:
Let's drop this entirely? It's redundant and the resulting screen is ugly.

Reply:
Agreed, I have deleted the `subheader` argument completely.

Included in the force-push, together with rebase on latest master to include the `CoinJoin` changes

![image](https://user-images.githubusercontent.com/42543243/198042457-5ce865af-8aa3-436a-948c-c6d988b85ed5.png)